tools/FakeItEasy.Deploy/FakeItEasy.Deploy.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Octokit" Version="0.31.0" />

```

Comment:
I just realized I used a more recent version than in `PrepareRelease` (upgraded because I couldn't find the "release" part of the API and I thought maybe it was not implemented in 0.26.0). If you prefer, I can upgrade PrepareRelease as well to be consistent.

Reply:
Oh, yeah. I commented around this before I saw your note.